app.json

```diff
@@ -28,7 +28,9 @@
       "supportsTablet": true,
       "infoPlist": {
         "NSCameraUsageDescription": "Diese App nutzt die Kamera um QR-Codes zu scannen.",
-        "NSPhotoLibraryUsageDescription": "Diese App nutzt die Medienbibliothek für die Auswahl eines Profilbildes."
+        "NSPhotoLibraryUsageDescription": "Diese App nutzt die Medienbibliothek für die Auswahl eines Profilbildes.",
+        "CFBundleLocalizations": ["de"],
+        "CFBundleDevelopmentRegion": "de_DE"
```

Comment:
i found the following:

> CFBundleLocalizations (Info.plist array specifying all supported localizations)
> CFBundleDevelopmentRegion (Info.plist value specifying the default language for the app)

https://docs.microsoft.com/en-us/xamarin/ios/app-fundamentals/localization/

default for `CFBundleDevelopmentRegion` seems to be "en_US". i think this key would be enough for our case, but let us keep the localizations as well.